src/functions-and-arrays.js

```diff
@@ -78,7 +228,21 @@ const wordsCount = [
   'matter'
 ];
 
-function howManyTimes() {}
+function howManyTimes(testWord, arr) {
```

Comment:
you need to pass array as first argument and test word as second argument.
 function howManyTimes( arr, testWord)